schema/definitions/0.8.0/examples/table_unsmry.yml

```diff
@@ -0,0 +1,175 @@
+    user:
+      id: jriv # $USER from ERT
+    description:
+      - yet other detailed description
+      - optional
+    restart_from: 15ce3b84-766f-4c93-9050-b154861f9100   # fmu.iteration.uuid for another iteration/case
+
+  iteration:
+    id: 0 # always an int, will be 0 for e.g. "pred"
+    uuid: 4b939310-34b1-4179-802c-49460bc0f799 # hash of case.uuid + iteration.id
+    name: "iter-0" # /"pred"
+
+  realization:
+    id: 33
+    uuid: 29a15b21-ce13-471b-9a4a-0f791552aa51
+    name: "realization-33"
+    parameters: # directly pass parameters.txt. This is potentially a lot of content, only a stub is included here.
+      SENSNAME: faultseal
+      SENSCASE: low
+      RMS_SEED: 1006
+      INIT_FILES:
+        PERM_FLUVCHAN_E1_NORM: 0.748433
+        PERM_FLUVCHAN_E21_NORM: 0.782068
+      KVKH_CHANNEL: 0.6
+      KVKH_US: 0.6
+      FAULT_SEAL_SCALING: 0.1
+      FWL_CENTRAL: 1677
+
+file:
+  relative_path: realization-33/iter-0/share/results/table/mysmry.arrow # case-relative
+  absolute_path: /some/absolute/path/realization-33/iter-0/share/results/table/mysmry.arrow
+  checksum_md5: kjhsdfvsdlfk23knerknvk23  # checksum of the file, not the data.
+  size_bytes: 5010144
+
+data: # The data block describes the actual data (e.g. surface). Only present in data objects
+
+  content: unsmry
+
+  # if stratigraphic, name must match the strat column. This is the official name of this surface.
+  name: myunsmry
+  stratigraphic: false
+
+  properties:
+  - name: DATE
+  - name: YEARS
+    unit: YEARS
+    is_total: false
+    is_rate: false
+    is_historical: false
+    keyword: YEARS
+    get_num: -32676
+  - name: FOPR
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: FOPR
+    get_num: 0
+  - name: FOPRH
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: true
+    keyword: FOPRH
+    get_num: 0
+  - name: FGPR
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: FGPR
+    get_num: 0
+  - name: RPR:20
+    unit: BARSA
+    is_total: false
+    is_rate: false
+    is_historical: false
+    keyword: RPR
+    get_num: 20
+  - name: GOPR:RFT
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: GOPR
+    wgname: RFT
+    get_num: 1
+  - name: WOPRH:R_A2
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: true
+    keyword: WOPRH
+    wgname: R_A2
+    get_num: 1
+
+  format: arrow
+  vertical_domain: depth # / time / null
+  depth_reference: msl # / seabed / etc
+  grid_model: # Making this an object to allow for expanding in the future
+    name: MyGrid # important for data identification, also important for other data types
+  spec: # class/layout dependent, optional? Can spec be expanded to work for all data types?
+    columns:
+      - YEARS
+      - FOPR
+      - FOPRH
+      - FGPR
+      - RPR:20
+      - GOPR:RFT
+      - WOPRH:R_A2
```

Comment:
Same here, need no make it more generic I think. It may be cases where `wells=["A", "B", "C"]` and `WOPRH:A` exists but not `WOPRH:B`, but we can consider the generic layout as a "all possible but not guaranteed" selection

Reply:
As previous... We may want to build something domain-specific/unsmry-specific, but that should imo be in next iterations.